neuraxle/steps/sklearn.py

```diff
@@ -66,13 +68,41 @@ def fit_transform(self, data_inputs, expected_outputs=None) -> ('BaseStep', Any)
             return self, self.wrapped_sklearn_predictor.predict(data_inputs)
         return self, self.wrapped_sklearn_predictor.transform(data_inputs)
 
+    def _sklearn_fit_transform_with_expected_outputs(self, data_inputs, expected_outputs):
+        if self.partial_fit:
+            self.wrapped_sklearn_predictor = self.wrapped_sklearn_predictor.partial_fit(data_inputs, expected_outputs)
+            out = self.wrapped_sklearn_predictor.transform(data_inputs)
+        else:
+            out = self.wrapped_sklearn_predictor.fit_transform(data_inputs, expected_outputs)
+        return out
+
+    def _sklearn_fit_transform_without_expected_outputs(self, data_inputs):
+        if self.partial_fit:
+            self.wrapped_sklearn_predictor = self.wrapped_sklearn_predictor.partial_fit(data_inputs)
+            out = self.wrapped_sklearn_predictor.transform(data_inputs)
+        else:
+            out = self.wrapped_sklearn_predictor.fit_transform(data_inputs)
+        return out
+
     def fit(self, data_inputs, expected_outputs=None) -> 'SKLearnWrapper':
         if expected_outputs is None or len(inspect.getfullargspec(self.wrapped_sklearn_predictor.fit).args) < 3:
-            self.wrapped_sklearn_predictor = self.wrapped_sklearn_predictor.fit(data_inputs)
+            self._sklearn_fit_without_expected_outputs(data_inputs)
         else:
-            self.wrapped_sklearn_predictor = self.wrapped_sklearn_predictor.fit(data_inputs, expected_outputs)
+            self._sklearn_fit_with_expected_outputs(data_inputs, expected_outputs)
         return self
 
+    def _sklearn_fit_with_expected_outputs(self, data_inputs, expected_outputs):
```

Comment:
I would have coded SKLearnWrapper.partial_fit and use an apply but it's ok like its it right now as well.